third_party/rust/crates/BUILD.bitflags-2.1.0.bazel

```diff
@@ -0,0 +1,47 @@
+###############################################################################
```

Comment:
If this PR eliminates dependencies on older versions of bitflags, let's also delete BUILD.bitflags-1.3.2.bazel andBUILD.bitvec-1.0.1.bazel!

Reply:
As above, these are transient dependencies that can't be removed.